types/chart.js/index.d.ts

```diff
@@ -198,6 +198,8 @@ declare namespace Chart {
         cutoutPercentage?: number;
         circumference?: number;
         rotation?: number;
+        // Plugins can require any options
+        plugins?: any;
```

Comment:
Plugins is a array of object that implements the plugin API : [doc](http://www.chartjs.org/docs/latest/developers/plugins.html#using-plugins)

Maybe you can add the `array` type with a plugin interface like [this](http://www.chartjs.org/docs/latest/developers/plugins.html#plugin-core-api) ? 😄

Reply:
Actually it should be `{ [plugin: string]: any }` I believe